src/components/Extensibility/components/FeaturedCard/FeaturedCard.js

```diff
@@ -38,17 +79,37 @@ export function FeaturedCard({ value, structure, schema, ...props }) {
     localStorage.setItem(hideBannerKey, updatedBoolean);
   };
 
-  return structure?.id && !hideBanner ? (
+  const isSAPuser = token => {
```

Comment:
This function should be registered as jsonata function, same like we have the rest of the functions in `jsonataWrapper`